cla_backend/apps/guidance/management/commands/load_markdown_files.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+import markdown
+import os
+import re
+import sys
+from django.core.management.base import BaseCommand
+
+from guidance.models import Note, Tag, NoteTagRelation
+
+meta_regex = re.compile('---\n(.*?)---\n', re.DOTALL)
+title_regex = re.compile('title: (.*?)\n')
+date_regex = re.compile('date: "(.*?)"\n')
+tag_regex = re.compile('  - (.*?)\n')
+
+
+class Command(BaseCommand):
+
+    help = 'Load Notes in from old markdown files'
+    requires_model_validation = True
+
+    def handle(self, *args, **options):
+
+        if len(args) != 1:
```

Comment:
Minor point, but maybe test for `len(args) < 1`. Otherwise you get an error if you pass any extra args.